### PR TITLE
Migrate from aphrodite to react-jss. Update MUI, react and react-dom versions.

### PR DESCRIPTION
**Pull request by sri2k1us**
- Note: I see some warning in storybook after migrating to JSS but I am not sure how to fix it yet. 
`data/BasicDetails/with test diskresource details error: ["Warning: Material-UI: the key `loadingStyle` provided to the classes property is not implemented in _class.
You can only override one of the following: "]
0: "Warning: Material-UI: the key `loadingStyle` provided to the classes property is not implemented in _class.
You can only override one of the following: `

### CODE REVIEW

File: react-components/src/data/details/BasicDetails.js [hunk 1]
```diff
@@ -52,7 +51,7 @@ function Md5(props) {
             <DEHyperlink text={getMessage("view")}/>
         </td>);
     } else {
-        return <td id={props.id} className={css(styles.value)}>{getMessage("folders")}</td>;
+        return <td id={props.id} className={props.classes.detailsValue}>{getMessage("emptyValue")}</td>;
```
Comment (psarando) [inline react-components/src/data/details/BasicDetails.js]: Did `getMessage("folders")` intentionally change to `getMessage("emptyValue")` here?

File: react-components/src/data/details/BasicDetails.js [hunk 2]
```diff
@@ -319,7 +324,9 @@ class BasicDetails extends Component {
                         tags={this.state.tags.values}
                         dataSource={this.state.dataSource}
                         onTagClick={this.handleTagClick}
-                        handleTagSelect={this.handleTagSelect}/>
+                        handleTagSelect={this.handleTagSelect}
+                    />
+                    <div>
```
Comment (psarando) [inline react-components/src/data/details/BasicDetails.js]: Is this extra `div` wrapper required around this `Dialog`?
If not, we should remove it; but if it is required, maybe the `Dialog` should be re-indented inside it.

File: react-components/src/data/details/BasicDetails.js (hunk 1 above)
Comment (sri2k1us) [reply react-components/src/data/details/BasicDetails.js]: It was a bug. So if MD5 is null or empty I should display empty value but it was displaying `folders` 🤦‍♂️

File: react-components/src/data/details/BasicDetails.js (hunk 2 above)
Comment (sri2k1us) [reply react-components/src/data/details/BasicDetails.js]: Yup those `<div>` needs to removed. I was trying to figure why `Dialog` was not working. Forgot to remove it once I figured out I need to update `react` and `react-dom`

File: react-components/package.json [hunk 3]
```diff
@@ -46,9 +45,9 @@
     "postcss-loader": "2.0.6",
     "promise": "8.0.1",
     "rc-tree": "^1.4.1",
-    "react": "^16.3.2",
+    "react": "^16.4.0",
     "react-dev-utils": "^4.1.0",
-    "react-dom": "^16.2.0",
+    "react-dom": "^16.4.0",
```
Comment (ash3rz) [inline react-components/package.json]: Should `aphrodite` be removed from package.json or are we still using it elsewhere?

Comment (sri2k1us) [reply react-components/package.json]: `aphrodite` can be removed from package.json

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**